project/build.properties

```diff
@@ -1,2 +1,2 @@
 // This file should only contain the version of sbt to use.
-sbt.version=0.13.18
+sbt.version=1.9.9
```

Comment:
for some reasons unknown spark does use sbt.version=1.9.3 
I don't know if there are any big changes there..

Reply:
Downgraded